### ISSUE TITLE
Struct Member Completion Issue

### ISSUE THREAD

**Issue by Daztek** [PhilippeChab/nwscript-ee-language-server#53]
Small issue with struct member completion, it only works when accessing the variable standalone, for example:

```nss
struct SomeRandomStruct
{
    int a;
    string b;
};
```

```nss
void SomeFunction(struct SomeRandomStruct str)
{
    str. // Completion shows the struct member vars here

    if (str. // Completion doesn't show the struct member vars
}
```

**Comment by PhilippeChab** [comment]
Good catch! Will into this.

**Comment by PhilippeChab** [comment]
Will be fixed in 2.1.0.
